rebar.config

```diff
@@ -16,14 +16,14 @@
 
 {deps, [
   {getopt, {git, "https://github.com/zmstone/getopt", {tag, "v1.0.2.1"}}},
-  {emqtt, {git, "https://github.com/emqx/emqtt", {tag, "1.11.0"}}},
+  {emqtt, {git, "https://github.com/emqx/emqtt", {tag, "1.13.1"}}},
   {prometheus, {git, "https://github.com/emqx/prometheus.erl", {tag, "v4.10.0.2"}}},
-  {cowboy, "2.9.0"}
+  {cowboy, "2.9.0"},
+  jsx
```

Comment:
I think it's best to pin a specific version to avoid surprises after a `jsx` new version is released.